ada/ast.py

```diff
@@ -8716,6 +8719,16 @@ class ExtendedReturnStmtObjectDecl(ObjectDecl):
     pass
 
 
+class NoTypeRenameObjectDecl(ObjectDecl):
+    """
+    Object declaration that is part of an iterator specification without
```

Comment:
Plz fix the docstring ;)

Reply:
Also please explain in this docstring why we had to introduce a new node type for this (at a high level), rather than just reuse the old one.